packages/blast-core/README.md

```diff
@@ -337,6 +338,69 @@ async function main () {
   // ...
 ```
 
+---
+## Creating a custom task
+
+Cudos Blast allows the creation of custom tasks that can easily run commonly used operations or help manage your workflow.
```

Comment:
The new lines (with one empty line) in this section are not consistent with the rest of the documentation. You can leave empty lines for completely changing the subject but I'm leaving it to you to decide when they are needed